sdk/python/feast/feature_store.py

```diff
@@ -176,3 +240,113 @@ def _get_requested_feature_views(
         feature_views_list.append(view)
 
     return feature_views_list
+
+
+def _run_reverse_field_mapping(
+    feature_view: FeatureView,
+) -> Tuple[List[str], List[str], str, Optional[str]]:
+    """
+    If a field mapping exists, run it in reverse on the entity names, feature names, event timestamp column, and created timestamp column to get the names of the relevant columns in the BigQuery table.
+
+    Args:
+        feature_view: FeatureView object containing the field mapping as well as the names to reverse-map.
+    Returns:
+        Tuple containing the list of reverse-mapped entity names, reverse-mapped feature names, reverse-mapped event timestamp column, and reverse-mapped created timestamp column that will be passed into the query to the offline store.
+    """
```

Comment:
formatter nit: not sure if we have a linter rule for line length but let's make it at least under 120 chars per line